sunsynk/definitions.py

```diff
@@ -165,13 +165,27 @@
 _SENSORS.append(
     SelectRWSensor(243, "Priority Mode", options={0: "Battery first", 1: "Load first"})
 )
+PROG1_TIME = TimeRWSensor(250, "Prog1 Time")
+PROG2_TIME = TimeRWSensor(251, "Prog2 Time", min=PROG1_TIME)
+PROG3_TIME = TimeRWSensor(252, "Prog3 Time", min=PROG2_TIME)
+PROG4_TIME = TimeRWSensor(253, "Prog4 Time", min=PROG3_TIME)
+PROG5_TIME = TimeRWSensor(254, "Prog5 Time", min=PROG4_TIME)
+PROG6_TIME = TimeRWSensor(255, "Prog6 Time", min=PROG5_TIME)
+PROG1_TIME.min = PROG6_TIME
```

Comment:
Are you sure prog1 should have a minimum of prog6? - have asked this question previously, without answer.